File: hello.go
package main

import "fmt"

func main() {
    messages := []string{"Hello", "World", "Go", "Loop", "Demo"}

    for i, msg := range messages {
        fmt.Printf("%d. %s\n", i+1, msg)
    }

    fmt.Println("\n逆序输出:")
    for i := len(messages) - 1; i >= 0; i-- {
        fmt.Printf("- %s\n", messages[i])
    }
}

File: hello.js
#!/usr/bin/env node


function main() {
    const messages = ["Hello", "World", "Node.js", "Loop", "Demo"];

    messages.forEach((msg, index) => {
        console.log(`${index + 1}. ${msg}`);
    });

    console.log("\n逆序输出:");
    for (let i = messages.length - 1; i >= 0; i--) {
        console.log(`- ${messages[i]}`);
    }
}


main();
